test/identical.js

```diff
@@ -26,4 +27,10 @@ describe('identical', function() {
     eq(R.identical(new Number(0), new Number(0)), false);
   });
 
+  it('perfect clones should be considered identical', function() {
+    fc.assert(fc.property(fc.dedup(fc.anything({ maxDepth: 0 }), 2), function(values) {
+      eq(R.identical(values[0], values[1]), true);
+    }));
+  });
```

Comment:
Can you explain this test? Identical shouldn't work for clones, as we have `equal` for that. What does `fc.dedup` do?

Reply:
`dedup` (badly named, I need to issue a version to rename it), is supposed to produce clones.

Indeed, it seems that I should move this test to equal instead 🤔 (and remove the `maxDepth` if equality is deep)

Reply:
I moved this test to equal and add another one to ensure that equal is symmetric. Regarding this newly added test, in the past I used it to find bugs on Jest (but it needed several runs as inputs causing bugs where pretty rare given the scope of inputs we authorize for this property).